docker-compose.yml

```diff
@@ -8,7 +8,7 @@ services:
     environment:
       POSTGRESQL_ADMIN_PASSWORD: mysecretpassword
   core:
-    image: fabric8-services/fabric8-wit:latest
+    image: fabric8/fabric8-wit:latest
```

Comment:
I would suggest to include `docker.io` in the image location. So it becomes: `docker.io/fabric8/fabric8-wit:latest`

Reply:
Thanks @baijum. `docker.io/fabric8/fabric8-wit:latest` would make more sense.